### PR TITLE
CIV-13989 Welsh translations for Claimant and Defendant LiP dashboard notifications - Judgments Online 1

### PR DESCRIPTION
**Pull request by sjamihmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-13989

### Change description ###
Welsh translations for Claimant and Defendant LiP dashboard notifications - Judgments Online 1

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by Omaira-Melo-Hmcts — APPROVED** [review]

File: src/main/resources/db/migration/V2024_05_24_1100__CIV-13792_judgment_by_admission_issued_defendant_scenario.sql [hunk 1]
```diff
@@ -16,9 +16,9 @@ INSERT INTO dbs.dashboard_notifications_templates ( template_name, title_En, tit
...
+        'Mae Dyfarniad wedi’i wneud yn eich erbyn',
+        '<p class="govuk-body">The judgment formalises the payment plan you’ve agreed with the claimant.<br>You’ve agreed to pay the claim amount of £${ccjDefendantAdmittedAmount} ${ccjPaymentMessageEn}. The first payment is due on ${ccjFirstRepaymentDateEn}.<br>The claimant’s details for payment and the full payment plan can be found on the judgment.<br>If you can no longer afford the repayments you’ve agreed with the claimant, you can <u>make an application to vary the judgment</u>.</p>',
+        '<p class="govuk-body">Mae’r dyfarniad yn ffurfioli’r cynllun taliadau yr ydych wedi cytuno arno gyda’r hawlydd.<br>Rydych wedi cytuno i dalu’ swm yr hawliad, sef £${ccjDefendantAdmittedAmount} ${ccjPaymentMessageCy}. Bydd y taliad cyntaf yn ddyledus ar ${ccjFirstRepaymentDateCy}.<br>Gellir dod o hyd i fanylion yr hawlydd ar gyfer talu a’r cynllun talu llawn ar y dyfarniad.<br>Os na allwch fforddio i dalu’r rhandaliadau rydych wedi cytuno arnynt gyda’r hawlydd, gallwch <u>wneud cais i amrywio’r dyfarniad</u>.</p>',
```
Comment (kalachandrasekar1) [inline src/main/resources/db/migration/V2024_05_24_1100__CIV-13792_judgment_by_admission_issued_defendant_scenario.sql:21]: Is this change cover both pay immediately and setByDate?

File: src/main/resources/db/migration/V2024_05_27_0900__CIV-13791_JudgmentOnlineRecordJudgmentDeterminationIssued.sql [hunk 2]
```diff
@@ -13,9 +13,9 @@ INSERT INTO dbs.dashboard_notifications_templates ( template_name, title_En, tit
...
+        '<p class="govuk-body">Mae’r rhandaliadau y mae’n rhaid i chi eu talu wedi cael eu penderfynu gan y llys. Bu iddynt wneud ‘penderfyniad ynghylch modd’ ac wedi penderfynu ar swm o randaliadau y gallwch eu fforddio. <br> ${paymentFrequencyMessageCy}.<br> Gellir dod o hyd i fanylion yr hawlydd ar gyfer talu a’r cynllun talu llawn a <a href="{VIEW_JUDGEMENT}" class="govuk-link">y dyfarniad.</a>.<br> Os ydych eisiau gwrthwynebu’r dyfarniad, neu ofyn i newid pryd a sut y byddwch yn talu swm yr hawliad yn ôl, gallwch <a href="{GENERAL_APPLICATIONS_INITIATION_PAGE_URL}" class="govuk-link"> wneud cais i roi’r dyfarniad o’r naill du (ei ddileu) neu amrywio’r dyfarniad.</a></p>',
```
Comment (kalachandrasekar1) [inline src/main/resources/db/migration/V2024_05_27_0900__CIV-13791_JudgmentOnlineRecordJudgmentDeterminationIssued.sql:18]: djDefendantNotificationMessage, djDefendantNotificationMessageCy - already having GA href url entries

Comment (sjamihmcts) [reply src/main/resources/db/migration/V2024_05_27_0900__CIV-13791_JudgmentOnlineRecordJudgmentDeterminationIssued.sql]: Updated the code ad discussed

File: src/main/resources/db/migration/V2024_05_24_1100__CIV-13792_judgment_by_admission_issued_defendant_scenario.sql (hunk 1 above)
Comment (sjamihmcts) [reply src/main/resources/db/migration/V2024_05_24_1100__CIV-13792_judgment_by_admission_issued_defendant_scenario.sql]: Updated the dashboard notifications for both pay immediately and setByDate

File: src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/dashboardnotifications/defendant/JudgementByAdmissionIssuedDefendantDashboardNotificationHandler.java [hunk 3]
```diff
@@ -42,7 +41,7 @@ public List<CaseEvent> handledEvents() {
     @Override
     public String getScenario(CaseData caseData) {
         if (caseData.isRespondent1LiP()) {
-            if (caseData.getDefenceAdmitPartPaymentTimeRouteRequired() == RespondentResponsePartAdmissionPaymentTimeLRspec.SUGGESTION_OF_REPAYMENT_PLAN) {
+            if (caseData.isLRvLipOneVOne() && null != caseData.getDefenceAdmitPartPaymentTimeRouteRequired()) {
```
Comment (ShwetaTandel-hmcts) [inline src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/dashboardnotifications/defendant/JudgementByAdmissionIssuedDefendantDashboardNotificationHandler.java:44]: Why have we changed this condition?

**Review by hmcts-version1-ignacio — APPROVED** [review]

**Review by Omaira-Melo-Hmcts — DISMISSED** [review]

**Review by ShwetaTandel-hmcts — DISMISSED** [review]

**Pull request merged**